src/main/java/io/github/jmh07/scoutingeyes/ScoutingEyes.java

```diff
@@ -82,6 +84,19 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
                     return false;
                 }
 
+                //If there is no recipient then the person executing the command is.
+                String recipient;
+                try {
+                    recipient = args[2];
+                    if(recipient == null){
+                        recipient=player.getName();
+                    }
+
```

Comment:
Moon only wants to give players this item by handing it out. Issues could arise if the full head of hair man accidentally gives it to the wrong person.  

Adding this recipient arg part may become useless. For now I would consider removing this.

Reply:
That is why I made it optional and so that he could use it if he wants to.